website/content/docs/k8s/multiport/reference/trafficpermissions.mdx

```diff
@@ -185,11 +185,11 @@ When [`spec.action`](#spec-action) _denies traffic_, Consul denies authorization
 #### Values
 
 - Default: None
-- Data type: Map
+- Data type: List of maps
```

Comment:
@ndhanushkodi , does this look good to you?

Reply:
If I look at a similar example in v1, it seems like this is more similar to a map: https://developer.hashicorp.com/consul/docs/connect/config-entries/service-router#routes-destination

Because there's only one `destinationRules`, and it specifies criteria for the destination service, using key value pairs, where the only key right now is portNames. So it seems more like a map to me.

Reply:
I'm not a yaml expert, but after some reading I think the difference between a list of map and map is

```yaml
      # destinationRules is a list of map
      destinationRules:
        - portNames: ["api-workload"]
```

```yaml
      # if destinationRules is a map
      destinationRules:
        portNames: ["api-workload"]
```

Please correct me if I am wrong. Thanks.

Reply:
Oh you're so right, I had been looking at this example [here](https://developer.hashicorp.com/consul/docs/k8s/multiport/reference/trafficpermissions#complete-configuration) which had the second case you mentioned in your comment, so I thought it was a map. But looking at the proto, and other examples on my machine, it is actually the first case in your comment

We may want to edit that example too to add the - before portNames in this PR too just to address it all at once

Reply:
Let me fix the examples as well.